### PR TITLE
StressChaos extension for more stress-ng options

### PR DESCRIPTION
**Pull request by huangwei2013**
### What problem does this PR solve? 

metions issue [#144 ](https://github.com/pingcap/chaos-mesh/issues/144), a experimental  intensify about **how to make use of hundreds stress-ng options** .
- It's not well tested, be careful pls
- chaos-mesh compile is a long job, be patient

### Check List 

Tests 

 - Unit test
 - E2E test
 - Manual test (add detailed scripts or steps below)
 - No code

Code changes

 - Has Go code change
 - Has CI related scripts change
 - Has Terraform scripts changeg

Side effects

 - Breaking backward compatibility

Related changes

 - Need to update the documentation

### Does this PR introduce a user-facing change?:

```release-note
NONE
```

### CODE REVIEW

**Comment by cwen0** [comment]
@huangwei2013 Thanks for your contribution, please sign CLA first.

File: api/v1alpha1/stresschaos_types.go [hunk 1]
```diff
@@ -225,13 +225,25 @@ func (in *Stressors) Normalize() (string, error) {
...
 		if in.CPUStressor.Load != nil {
 			stressors += fmt.Sprintf(" --cpu-load %d",
 				*in.CPUStressor.Load)
 		}
+
+		if in.CPUStressor.Options != nil {
+			for _,v := range in.CPUStressor.Options{
+				stressors += fmt.Sprintf(" %v ", v)
+			}
+		}
```
Comment (Yisaer) [inline api/v1alpha1/stresschaos_types.go:246]: What about if user set the `--cpu-load ` option? How would we merge the adding option and the existed option.

Comment (huangwei2013) [reply api/v1alpha1/stresschaos_types.go:246]: @Yiaser  by stress-ng itself will take over the repeated options ( tested: later one will replace former one)

A test case is : （ --cpu is a more clear test case, Top to find there is only on  Processor of "stress-ng-cpu "）
`stress-ng --cpu-load 10 --cpu 2 --cpu  1`

**Comment by zhouqiang-cl** [comment]
@cwen0 @fewdan please take a look at this CI

**Comment by fewdan** [comment]
@huangwei2013 
Sorry, the code you submitted caused the CI failed.  It seems that you have added a new "Options" field in stresschaos, but there is no such field in "stresschaos_types_test.go". So, can you modify "stresschaos_types_test.go" to make the code pass CI test? Thank you.
image

**Pull request merged**